### PR TITLE
fix tests

### PR DESCRIPTION
**Pull request by dexhorthy**

### CODE REVIEW

**Review by laverya — APPROVED** [review]

File: cli/test/kots_app_test.go [hunk 1]
```diff
@@ -93,34 +93,13 @@ var _ = Describe("kots apps", func() {
...
-			var stdout bytes.Buffer
-			var stderr bytes.Buffer
-
-			rootCmd := cmd.GetRootCmd()
-			rootCmd.SetArgs([]string{"app", "create", newName})
-
-			err = cmd.Execute(rootCmd, nil, &stdout, &stderr)
-			req.NoError(err)
-
-			req.Empty(stderr.String(), "Expected no stderr output")
```
Comment (laverya) [inline cli/test/kots_app_test.go:113]: IMO these need to get included after the 'app create' below

**Review by laverya — CHANGES_REQUESTED** [review]
when combining the tests the new one should have the same checks as the old

File: cli/test/kots_app_test.go (hunk 1 above)
Comment (dexhorthy) [reply cli/test/kots_app_test.go:113]: good call thank you

**Review by laverya — APPROVED** [review]
looks good to me - reconciles tests with new output format and combines two tests to cover the same code but be more consistent

**Pull request merged**